frontend/plan/pages/index.tsx

```diff
@@ -132,7 +146,15 @@ function Index() {
     const scrollTop = () => window.scrollTo(0, 0);
     const isExpanded = view === 1;
 
-    const [showLoginModal, setShowLoginModal] = useState<boolean>(true);
+    // cart vs alerts
+    const [showAlerts, setShowAlerts] = useState(false);
+    useEffect(() => {
+        const hash = router.asPath.split("#")[1];
+        setShowAlerts(hash === "alerts");
+    }, [setShowAlerts]);
```

Comment:
Unacceptable.

Reply:
Index page will be rerendered anyways; and you want to extract the hash router value as the component is rendering(or else user will see a page with the incorrect state). Set the default state value to depend on router instead.